### ISSUE TITLE
validation_app openvino R5 produces 1.02% top1 accuracy for FP32 inception_v4

### ISSUE THREAD

**Issue by jetsmith** [opencv/dldt#132]
hello, i have tried to calculate fp32 accuracy using Openvino R5 "validation_app" but it produced weird results:
[ INFO ] Average infer time (ms): 64.45 (15.52 images per second with batch size = 1)                   │classification_sample_async  InferenceEngineUnitTests         speech_sample
Top1 accuracy: 1.02% (51 of 5000 images were detected correctly, top class is correct)                  
Top5 accuracy: 16.94% (847 of 5000 images were detected correctly, top five classes contain required 

Well , i think the results should be like this:
top1 accuracy: 80.2%(it is tested using tensorflow).

Here is my run condition:
**OS**: Ubuntu 16.04
**project version**: Openvino R5(2019 branch)
**pretrained model**: **inception_v4 tensorflow model** [https://github.com/tensorflow/models/tree/master/research/slim](url)
**validation dataset** : imagenet 2012(i use one tenth of total images and 5 images each class)

Here are my experimental steps:

**1. Converting a TensorFlow Model**
under the dldt/model-optimizer, run mo_tf.py, the following is my sh script:

> python3 mo_tf.py \                                                                                                                                                                                    
>          --input_model  ./inception_v4.pb \                                                                                                                   
>          --model_name Inception_v4 \                                                                                                                                                                     
>          --output_dir inception_v4 \                                                                                                                                                                     
>          --input_shape [1,299,299,3] \                                                                                                                                                                   
>          --data_type FP32 \                                                                                                                                                                              
>          --mean_value [127.5,127.5,127.5] \                                                                                                                                                              
>          --scale 127.5 \                                                                                                                                                                                 
>          --reverse_input_channels \                                                                                                                                                                      
>          --input input_img \                                                                                                                                                                             
>          --output InceptionV4/Logits/Logits/BiasAdd

output:
Model Optimizer arguments:
Common parameters:
        - Path to the Input Model:      ****inception_v4.pb
        - Path for generated IR:        ****/inception_v4
        - IR output name:       Inception_v4
        - Log level:    ERROR
        - Batch:        Not specified, inherited from the model
        - Input layers:         input_img
        - Output layers:        InceptionV4/Logits/Logits/BiasAdd
        - Input shapes:         [1,299,299,3]
        - Mean values:  [127.5,127.5,127.5]
        - Scale values:         Not specified
        - Scale factor:         127.5
        - Precision of IR:      FP32
        - Enable fusing:        True
        - Enable grouped convolutions fusing:   True
        - Move mean values to preprocess section:       False
        - Reverse input channels:       True
TensorFlow specific parameters:
        - Input model in text protobuf format:  False
        - Path to model dump for TensorBoard:   None
        - List of shared libraries with TensorFlow custom layers implementation:        None
        - Update the configuration file with input/output node names:   None
        - Use configuration file used to generate the model with Object Detection API:  None
        - Operations to offload:        None
        - Patterns to offload:  None
        - Use the config file:  None
Model Optimizer version:        unknown versio

it generate three files:
Inception_v4.bin  Inception_v4.mapping  Inception_v4.xml

**2. run validation_app** 
first build the project under dldt/inference-engine/, then validation_app binaries could be found in inference-engine/bin/intel64/Release, run the shell like this:
```
./validation_app -t C \                                                                                                                                                                                 
          -i ***imagenet_val/5000_image/img_class.txt \                                                                                                                         
          -m ****/dldt/model-optimizer/inception_v4/Inception_v4.xml \                                                                                                            
          -d CPU \                                                                                                                                                                                        
          -ppType ResizeCrop \                                                                                                                                                                            
          -ppSize 299     
```                                                                                                                                                                               
  
my image list formats as: 
  ILSVRC2012_val_00005901.JPEG 97                                                                                                                                                                        
  ILSVRC2012_val_00006460.JPEG 97                                                                                                                                                                        
  ILSVRC2012_val_00001602.JPEG 406                                                                                                                                                                       
  ILSVRC2012_val_00002523.JPEG 406                                                                                                                                                                       
  ILSVRC2012_val_00002876.JPEG 406   

well, 5000 images are placed under imagenet_val/5000_image. 
i have checked all my steps and confused about why the top1 acc is only **1.02%**!!!!!!

**Comment by elvin-nnov** [comment]
There is quite common approach for adding one more background 0th class to TF classification models. Validation app has a special option dedicated for this case:
```
    Classification-specific options:
      -Czb true               "Zero is a background" flag. Some networks are trained with a modified dataset where the class IDs  are enumerated from 1, but 0 is an undefined "background" class (which is never detected)
```
Another note - to reproduce numbers published together with network, you need to be very careful and point correct mean values and scales in Model Optimizer commandl line during conversion of the model

**Comment by jetsmith** [comment]
@elvin-nnov i have solved it, many thanks！

**Comment by CraneLu** [comment]
> @elvin-nnov i have solved it, many thanks！

Could you tell us how to solve it?Where is problem?I got similar problem.
Thanks!

**Comment by busyyang** [comment]
is the `validation_app` departched in laster version? I cann't find the codes for validation_app in `OpenVINO 2020.02` and  `OpenVINO 2020.03`.

**Comment by elvin-n** [comment]
@busyyang it was substituted by accuracy checker staying in https://github.com/openvinotoolkit/open_model_zoo/tree/master/tools/accuracy_checker

**Comment by busyyang** [comment]
> @busyyang it was substituted by accuracy checker staying in https://github.com/openvinotoolkit/open_model_zoo/tree/master/tools/accuracy_checker

@elvin-n thanks, it is help a lot.
